pkg/account/README.md

```diff
@@ -0,0 +1,78 @@
+# Create
+
+Request
+
+```json
+{
+    "id": 1234567,  // uint
```

Comment:
json does not accept comments. Consider moving it some where else, like mapping it to a go struct.